Makefile

```
@@ -489,10 +493,31 @@ verify-links:
 	fi;
 .PHONY: verify-links
 
-verify: verify-gofmt verify-codegen verify-crds verify-helm-schemas verify-helm-charts verify-deploy verify-examples verify-govet verify-helm-lint verify-links
+# $1 - client name
+# $2 - source file
+# $3 - client directory
+define generate-swagger-client
+	@rm -Rf $(3) && mkdir -p $(3)
+	$(SWAGGER) generate client -A $(1) -f $(2) -t $(3) &> /dev/null
```

```suggestion
	$(SWAGGER) generate client -A scylla-manager -f $(2) -t $(3) &> /dev/null
```
This is the common part, it shouldn't be parametrized

---

Right, fixed it.